python/pyarrow/tests/test_dataset_encryption.py

```diff
@@ -151,3 +155,61 @@ def test_write_dataset_parquet_without_encryption():
 
     with pytest.raises(NotImplementedError):
         _ = pformat.make_write_options(encryption_config="some value")
+
+
+@pytest.mark.skipif(
+    not encryption_unavailable, reason="Parquet Encryption is currently enabled"
```

Comment:
Shouldn't this be skipped if encryption is unavailable rather than the other way around?
```suggestion
    encryption_unavailable, reason="Parquet Encryption is not currently enabled"
```